oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/test/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/mysql/MySQLTableInstallerTest.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.skywalking.oap.server.storage.plugin.jdbc.mysql;
+
+import org.apache.skywalking.oap.server.core.source.DefaultScopeDefine;
+import org.apache.skywalking.oap.server.core.source.Event;
+import org.apache.skywalking.oap.server.core.storage.annotation.Storage;
+import org.apache.skywalking.oap.server.core.storage.model.Model;
+import org.apache.skywalking.oap.server.core.storage.model.StorageModels;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author jayxu
+ */
+public class MySQLTableInstallerTest {
+    @Test
+    public void test() throws Exception {
+        new DefaultScopeDefine.Listener().notify(Event.class);
+        Model m = new StorageModels().add(Event.class, DefaultScopeDefine.EVENT,
+            new Storage(null, false, null), false);
+
+        m.getColumns().stream()
+            .filter(c -> Event.PARAMETERS.equals(c.getColumnName().getName()))
+            .findAny().ifPresent(c -> {
+                String sql = new TestTableInstaller().getColumn(c);
+//                System.out.println(sql);
+                Assert.assertTrue("Should contains 'TEXT'",
+                    sql.contains("TEXT"));
```

Comment:
I think we have a type mapping method, don't have to test in a separate UT.

Reply:
`MySQLTableInstaller#getColumn` is the source of column type. If you want to add a UT, add it for MySQLTableInstaller.